adyenv6b2ccheckoutaddon/src/com/adyen/v6/facades/DefaultAdyenCheckoutFacade.java

```diff
@@ -676,7 +681,11 @@ private String getShopperLocale() {
             String[] keyValuePair = parameter.split("=");
             String key = keyValuePair[0];
             String value = request.getParameter(key);
-            queryParameters.put(key, value);
+
+            // Add only HPP parameters for signature calculation
+            if (HPP_RESPONSE_PARAMETERS.contains(key) || key.startsWith("additionalData.")) {
+                queryParameters.put(key, value);
+            }
         }
```

Comment:
why making this restriction see comment above as well.

Reply:
see reply above